packages/adapters/watcher/src/alert/pagerduty.ts

```diff
@@ -0,0 +1,74 @@
+import { trigger } from "@pagerduty/pdjs";
+
+import { Report } from "../types";
+
+const ROUTING_KEY = "xxx";
+
+export const alertViaPagerDuty = async (report: Report) => {
+  const {
+    timestamp,
+    event,
+    reason,
+    errors,
+    logger,
+    requestContext,
+    methodContext,
+    domains,
+    relevantTransactions,
+    rpcs,
+  } = report;
+
+  logger.info("Sending message to pager duty", requestContext, methodContext, {
+    timestamp,
+    event,
+    reason,
+    errors,
+    domains,
+    relevantTransactions,
+    rpcs,
+  });
+
+  if (ROUTING_KEY.length != 32) {
+    throw new Error("PagerDuty Routing Key is invalid");
+  }
+
+  try {
```

Comment:
This try catch shouldn't be here. We should throw an error in this function and we should catch it at the top level `alert` function.